packages/react/src/generators/application/lib/normalize-options.ts

```diff
@@ -29,6 +29,7 @@ export function normalizeOptions(
   assertValidStyle(options.style);
 
   options.routing = options.routing ?? false;
+  options.strict = options.strict ?? false;
```

Comment:
Perhaps strict should fallback to true?

Reply:
Good point. I think we need an advice from @jaysoo what is better for fallback.

Reply:
@puku0x Yes, let's use strict by default, so users will need to opt out with `--no-strict`, `--strict false` or set it in their `nx.json` file.

Reply:
Strict mode by default would be better so we dont change current behavior.